python/paddle/fluid/framework.py

```diff
@@ -64,7 +64,7 @@
 CONTROL_DEP_VAR_PREFIX = core.kControlDepVarName()
 
 _dygraph_tracer_ = None
-_dygraph_current_expected_place_ = None
+_global_expected_place_ = None
 _current_device = None
```

Comment:
remove `_current_device`, and make `device_guard` use `_set_expected_place` internally.

Reply:
device_guard is not the global place change, if we use _set_expected_place will change the device_gurad function

Reply:
It is recommended to change the device_guard in the next PR. If something goes wrong, it’s better to deal with it.